.github/workflows/post_merge.yml

```diff
@@ -0,0 +1,34 @@
+name: Post PR Merge
+
+on:
+  pull_request:
+    # Patterns matched against refs/heads
+    branches:
+      - release/next
+    types: [ closed ]
+
+jobs:
+  release_merge:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git - Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Install cz
+        run: pip install --user -U Commitizen
+      - name: Record release version
+        run: |
+          echo "RELEASE_VERSION=$(cz version --project)" >> $GITHUB_ENV
+      - name: Tag release
+        uses: tvdias/github-tagger@v0.0.1
```

Comment:
[nitpick]: Maybe we should consider using some more maintained action?

Reply:
I changed this to manually tag. The local git identity should be set correctly having checked out the repo using the RELEASE_ACCESS_TOKEN